design/Limits.md

```diff
@@ -0,0 +1,29 @@
+# GPULimits Explainer
+
+This document lists the citations for the "limits" in the WebGPU API that decide the minimum capabilities of a compliant WebGPU implementation.
+
+## The GPULimits Dictionary (10/29/19)
+
+```javascript
+dictionary GPULimits {
+    unsigned long maxBindGroups = 4;
+    unsigned long maxDynamicUniformBuffersPerPipelineLayout = 8;
+    unsigned long maxDynamicStorageBuffersPerPipelineLayout = 4;
+    unsigned long maxSampledTexturesPerShaderStage = 16;
+    unsigned long maxSamplersPerShaderStage = 16;
+    unsigned long maxStorageBuffersPerShaderStage = 4;
+    unsigned long maxStorageTexturesPerShaderStage = 4;
+    unsigned long maxUniformBuffersPerShaderStage = 12;
+};
+```
+
+Limit | API Doc | gpuweb issue/PR
+--- | --- | ---
+`maxBindGroups = 4;` | [Vulkan](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/html/vkspec.html#limits-minmax) `maxBoundDescriptorSets` |
```

Comment:
unfortunate that Vulkan spec is so heavy

Reply:
I thought it would at least redirect to the specific section. :(

Reply:
It does for me, if I don't touch it while it's loading.

How about the LunarG chunked version?
https://vulkan.lunarg.com/doc/view/latest/linux/chunked_spec/chap36.html#limits-minmax

Reply:
https://vulkan.lunarg.com/doc/view/latest/linux/chunked_spec/chap36.html#limits-required